tests/tree_inference/test_logprob.py

```diff
@@ -161,7 +163,7 @@ def test_logprobability_fn():
     expected = 5 * jnp.log(4 * jnp.exp(3 * jnp.log(0.5)))
 
     # define logprob fn
-    log_prob = logprob.logprobability_fn(mutation_matrix, tree, theta)
+    log_prob = logprob.logprobability_fn(mutation_matrix, tree, theta)  # type: ignore
```

Comment:
Note all tests of `log_prob` functions ignore the types of trees inputted now as a consequence of enforcing the `OrderedTree`s